File: 4º Periodo/Analise de Algoritmos/troco.c
#include <stdio.h>

int main(){
    int moedas[5] = {1,5,10,25,50};
    int valor = 74;
    int index = 5;
    while (valor != 0 )
    {
        if(valor - moedas[index] >= 0){
            printf("Valor: %i | Moeda de Troco: %i\n", valor, moedas[index]);
            valor = valor - moedas[index];
        }else{
            index--;
        }
    }
    printf("%i\n", valor);
}

File: 4º Periodo/Analise de Algoritmos/Atividade 9/troco.go
package main

import "fmt"

func main() {

	var moedas = []int{1, 5, 10, 25, 50}
	in := 74
	index := 4
	for in >= 0 {
		if in-moedas[index] >= 0 {
			in -= moedas[index]
		} else {
			index -= 1
		}
	}

	fmt.Println(in)
}
